packages/core/src/metadata/operator/standard/supersetOf.ts

```diff
@@ -13,12 +12,11 @@ export const supersetOfOperator: MetadataOperator = {
   ],
   operatorJsonValues: [
     z.array(z.string()),
-    z.array(z.record(z.string().or(z.number()), z.unknown())),
+    // TODO: See how we want to we handle the comparison of objects
+    // z.array(z.record(z.string().or(z.number()), z.unknown())),
```

Comment:
We should also remove it from parameterJsonValues then? or will it work if the parameter is an object?

Reply:
Maybe open an issue?